internal/sops/pgp/keysource.go

```diff
@@ -332,7 +332,7 @@ func (key *MasterKey) passphrasePrompt() func(keys []openpgp.Key, symmetric bool
 		conn, err := gpgagent.NewConn()
 		if err == gpgagent.ErrNoAgent {
 			fmt.Print("Enter PGP key passphrase: ")
-			pass, err := gopass.GetPasswd()
+			pass, err := term.ReadPassword(int(os.Stdin.Fd()))
```

Comment:
I think we should error out here, there is no stdin available in the controller. This way we can get rid of x/term.

Reply:
This was originally copied over to maintain full compatibility with SOPS' way of working, while breaking free from the user (environment) specific assumptions in upstream.

But I do agree dropping the dependency is better.

Reply:
Thanks, guys. I was actually going to ask you about it since I was confused about how to test this and I noticed in the docs it is explicitly mentioned that only passphrase-less keys are supported. Removing the dependency makes perfect sense in this case. :ok_hand: